docs/buildx.md

```diff
@@ -0,0 +1,51 @@
+# Running Buildx
+
+Docker Buildx is a CLI plugin that extends the docker command with the full support of the features provided by Moby BuildKit builder toolkit. BuildKit is designed to work well for building for multiple platforms and not only for the architecture and operating system that the user invoking the build happens to run. More information can be found [docker-buildx](https://docs.docker.com/buildx/working-with-buildx/)
+
+### Enable docker experimental client flag
```

Comment:
Could you please double-check this section. 
When I followed these guidelines, the docker experimental was not enabled. 
So, I managed to work around this by explicitly specifying the config file using the following command
`docker --config ~/.docker/`
I think this is due to `$DOCKER_CONFIG` is not set by default in some installations.

Reply:
I have specified the instructions mentioned here https://docs.docker.com/engine/reference/commandline/cli/#experimental-features
and installed docker using this method
https://docs.docker.com/engine/install/ubuntu/#install-using-the-convenience-script

I have only tested the above method on Ubuntu OS and seems to work fine. As you have mentioned it's not working in some installations, should we add another helpful link/instructions?

Reply:
I used centOS 7 for building this image. Docker was installed from the standard repository 
```
sudo apt-get install docker-ce docker-ce-cli containerd.io
```
Also, I uninstalled the docker an installed docker again with the convenience-script you mentioned. Now, I could able to enable the experimental features of the client as you mentioned.
This is the example I followed to enable the experimental feature in a standard installation.
https://docs.docker.com/engine/swarm/configs/#simple-example-get-started-with-configs